packages/react-components/react-charts-preview/src/components/HorizontalBarChart/HorizontalBarChart.tsx

```diff
@@ -45,20 +46,27 @@ export const HorizontalBarChart: React.FunctionComponent<IHorizontalBarChartProp
   const [barCalloutProps, setBarCalloutProps] = React.useState<IChartDataPoint>();
   const [callOutAccessibilityData, setCallOutAccessibilityData] = React.useState<IAccessibilityProps | undefined>();
   const [barSpacingInPercent, setBarSpacingInPercent] = React.useState<number>(0);
+  const [isPopoverOpen, setPopoverOpen] = React.useState<boolean>(false);
+  const [clickPosition, setClickPosition] = React.useState({ x: 0, y: 0 });
 
   function _refCallback(element: SVGGElement, legendTitle: string | undefined): void {
     _refArray.push({ index: legendTitle, refElement: element });
   }
 
-  function _hoverOn(hoverValue: string | number | Date | null, point: IChartDataPoint): void {
+  function _hoverOn(
+    event: React.MouseEvent<SVGRectElement, MouseEvent>,
+    hoverVal: string | number | Date | null,
```

Comment:
can you remove null as possibility if you need to always add a non nullable reference using !

Reply:
sure, this was already there, didn't change